### PR TITLE
Fixes

### PR DESCRIPTION
**Pull request by atifsaddique211f**

### CODE REVIEW

File: src/app/mail/mail-list/mail-folder/generic-folder/generic-folder.component.html [hunk 1]
```diff
@@ -0,0 +1,240 @@
...
+    <table class="table table-inbox">
+      <tbody>
+      <tr class="mail-list-row" *ngFor="let mail of mails; let i=index"
+          [ngClass]="{'is-unread': !mail.read, 'is-read': mail.read, 'is-selected':mail.marked}">
+
+
+        <!-- Check box -->
+        <td class="mail-checkbox pl-4">
+          <div class="fancy-field-group">
+            <input class="d-none fancy-field-control" id={{mail.id}} type="checkbox" [(ngModel)]="mail.marked"/>
```
Comment (Waseem826) [inline src/app/mail/mail-list/mail-folder/generic-folder/generic-folder.component.html]: Missing quotes around value of `id`

File: src/app/mail/mail-list/mail-folder/generic-folder/generic-folder.component.html [hunk 2]
```diff
@@ -0,0 +1,240 @@
...
+            <label for={{mail.id}} class="fancy-field-empty"></label>
```
Comment (Waseem826) [inline src/app/mail/mail-list/mail-folder/generic-folder/generic-folder.component.html]: Same here for `for`

File: src/app/mail/mail-list/mail-folder/generic-folder/generic-folder.component.html (hunk 1 above)
Comment (atifsaddique211f) [reply src/app/mail/mail-list/mail-folder/generic-folder/generic-folder.component.html]: @Waseem826 I just moved this code, this is written by zain.
Secondy I think it is allowed without quotes.

File: src/app/mail/mail-list/mail-folder/generic-folder/generic-folder.component.html (hunk 2 above)
Comment (atifsaddique211f) [reply src/app/mail/mail-list/mail-folder/generic-folder/generic-folder.component.html]: it is allowed.

**Pull request merged**